data.sql

```diff
@@ -3,4 +3,41 @@
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
```

Comment:
[ OPTIONAL BUT RECOMMENDED ] Please always create a new feature branch for every new feature and create a new Pull request from that new branch.

Reply:
Thanks...